index.html

```diff
@@ -67,13 +68,237 @@ <h2>.</h2>
                 </li>
             </ul>         
         </section>
-    <div>
+    </section>
+    <section>
+        <h6 class="hidden">.</h6>
         <div class="elm_1">.</div>
         <div class="elm_2">.</div>
         <div class="elm_3">.</div>
         <div class="elm_4">.</div>
         <div class="elm_5">.</div>
         <div class="elm_6">.</div>
-    </div>
+    </section>
+    <section class="work_grid">
+    <ul>
+        <li class="box">
+            <div class="prev_1 preview item"></div>
+            <div class="title item">
+                <h1>Tonic</h1>
+            </div>
+            <ul class="features item">
+                <li><h4 class="feature_1">CANOPY</h4></li>
+                <li><div class="dot"></div></li>
+                <li><h4 class="feature_2">Back End Dev</h4></li>
+                <li><div class="dot"></div></li>
+                <li><h4 class="feature_2">2015</h4></li>
+            </ul>
+            <div class="description item">
+                <p>
+                    A daily selection of privately perzonalized reads; no account or sign-ups required.
+                </p>
+            </div>
+            <ul class="languages item">
+                <li><h5>html</h5></li>
+                <li><h5>css</h5></li>
+                <li><h5>JavaScript</h5></li>
+            </ul>
```

Comment:
- [ ] _[Optional]_ In this section, you're required to use the font and colours (background and font) defined for the template. 

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/120880396-31841880-c5c2-11eb-9dbc-9b63e1e49344.png) | ![image](https://user-images.githubusercontent.com/16058470/120880412-4660ac00-c5c2-11eb-97ad-f8ab62d8e4f6.png)